timesys/high-level-events.py

```diff
@@ -14,7 +14,8 @@
     'TunSI': 'Evt09', 'FOFBS': 'Evt10',
 
     'Dsbld':  'Evt00',
-    'Intlk': 'Evt117', 'RFKll': 'Evt124',
+    'Intlk': 'Evt117', 'ItlkR': 'Evt118',
```

Comment:
Slightly prettier, maybe? `RItlk` for "Redundant Interlock"

Reply:
could be. Let's see others opinion on this. But it seems putting the R before is a bit prettier indeed.

Reply:
I think I prefer "Interlock Redundancy"